forge-gui/res/cardsfolder/upcoming/inquisitor_eisenhorn.txt

```diff
@@ -0,0 +1,13 @@
+Name:Inquisitor Eisenhorn
+ManaCost:2 U B
+Types:Legendary Creature Human Inquisitor
+PT:2/3
+T:Mode$ Drawn | ValidCard$ Card.YouOwn | Number$ 1 | OptionalDecider$ You | Static$ True | ForReveal$ True | Execute$ DBReveal | TriggerZones$ Battlefield | TriggerDescription$ You may reveal the first card you draw each turn as you draw it. Whenever you reveal an instant or sorcery card this way, create Cherubael, a legendary 4/4 black Demon creature token with flying.
+SVar:DBReveal:DB$ Reveal | Defined$ You | RevealDefined$ TriggeredCard | RememberRevealed$ True | SubAbility$ DBToken
```

Comment:
No reason to template this differently than God-Eternal Kefnet by leaving out the real trigger there